tasks/bootstrap-environment-variables.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
How this environment variables will exported to the system?

Reply:
On the location defined on the default variables, they won't, they need to be loaded by another mechanism, like by a systemd service file. It's also easy to change it to `/etc/environment` or any other location.

Reply:
Forgot to mention, /etc/environment is not set by default because it's world readable, and as a consequence, having credentials as environment variables is not as secure as having the environment variables file on the projects shared folder, so only the user that runs the app can read the values.

Reply:
OK sounds good,  but in case the you don't want use environment variables it is posable to disable this?

Reply:
Was not, but I just added the conditional. Only when defining the `sb_debian_base_env_vars` dictionary, the task will run.